source/slang/slang-ast-support-types.h

```diff
@@ -128,6 +128,13 @@ namespace Slang
         kConversionCost_Impossible = 0xFFFFFFFF,
     };
 
+    typedef unsigned int BuiltinConversionName;
+    enum : BuiltinConversionName
+    {
+        kBuiltinConversion_Unknown = 0,
+        kBuiltinConversion_FloatToDouble = 1,
```

Comment:
Again nitpicky... 

The use of `Name` and `Context` seem like they are a little confusing as both have more normal programming meanings, and in Slang specifically. A context being a bunch of state, and Name being a uniquified string or at least text related.

CoercionContext  -> CoercionSite/CoercionLocation?
BuiltinConversionName -> Has prefix of just BuiltinConversion, but Kind, Style etc also work.